config/config_test.go

```diff
@@ -3190,64 +3190,66 @@ func TestTCF2FeatureOneVendorException(t *testing.T) {
 }
 
 func TestIsAccountEventEnabled(t *testing.T) {
+	boolFalse := false
+	boolTrue := true
 	tests := []struct {
 		name                    string
 		deprecatedEventsEnabled *bool
-		eventsEnabled           bool
+		eventsEnabled           *bool
 		expectedIsEventEnabled  bool
 	}{
 		{
 			name:                    "events disabled, events.enabled false",
 			deprecatedEventsEnabled: boolPointer(false),
-			eventsEnabled:           false,
+			eventsEnabled:           &boolFalse,
 			expectedIsEventEnabled:  false,
 		},
 		{
 			name:                    "events disabled, events.enabled true",
 			deprecatedEventsEnabled: boolPointer(false),
-			eventsEnabled:           true,
+			eventsEnabled:           &boolTrue,
```

Comment:
Nitpick: Can we simplify by using the `ToPtr()` function from `util/ptrutil/ptrutil.go`?
```
3209 -           deprecatedEventsEnabled: boolPointer(false),
3210 -           eventsEnabled:           &boolTrue,
     +           deprecatedEventsEnabled: ptrutil.ToPtr(false),
     +           eventsEnabled:           ptrutil.ToPtr(true),
```